api/resource.rb

```diff
@@ -219,6 +219,18 @@ def gettable_properties
       all_user_properties.reject(&:url_param_only)
     end
 
+    # Returns the list of top-level properties once the object has been
```

Comment:
nit: "once the object has been collapsed of any `flatten_object` properties." feels like you want to say "once any nested objects with `flatten_object` set to true have been collapsed"